guide/payments/introduction.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: guide
+title: Payments
+description: Diving into the user experience of moving Bitcoin
+nav_order: 4
+has_children: true
```

Comment:
You can remove `has_children` or set it to false for now. Enabling this adds makes the "Payments" nav option expandable, but there are no sub-pages right now.

Reply:
```suggestion
has_children: false
```